Adding MySQL to the documentation (#515)
mattermost/focalboard: website/site/content/download/personal-edition/ubuntu.md

"dbconfig": "postgres://boardsuser:boardsuser-password@localhost/boards?sslmode=disable&connect_timeout=10",
```

## Install MySQL 

As an alternative to Postgres, you also can store your data in a MySQL/MariaDB database.
To install, run:

```
sudo apt install mariadb-server mariadb-client
```

Log in as `root` in your database:
```
sudo mysql 
```

At the MySQL prompt, run the following commands (change `user/password` to your own values):
```
CREATE DATABASE boards;
GRANT ALL on boards.* to <b>'boardsuser'@'localhost'</b> identified by '<b>boardsuser-password</b>';
```

Exit the mysql-prompt:
```
exit
```

Edit the Focalboard `config.json`:

```
nano /opt/focalboard/config.json
```

Change the dbconfig setting to use the MySQL database you created:

> "dbtype": "mysql",

Reviewer: Here are missing the 3 backticks

"dbconfig": "boardsuser:boardsuser-password@tcp(127.0.0.1:3306)/boards",

> ```,

Reviewer: And I think this `,` here is not necesary, also the code block is going to be there and the `,` is going to be in the next line, I would remove that.

"dbconfig": "postgres://boardsuser:boardsuser-password@localhost/boards?sslmode=disable&connect_timeout=10",
```

## Install MySQL 

> Alternate to Postgress you also can store your data in a MySQL/MariaDB database.

Reviewer 2: ```suggestion
As an alternative to Postgres, you also can store your data in a MySQL/MariaDB database.
```

> 16). To install, run:

Reviewer: What is this `16).` here?

Author: Some copy-paste error.

```
sudo apt install mariadb-server mariadb-client
```

> Log in as root in your database:

Reviewer 2: ```suggestion
Log in as `root` in your database:
```

```
sudo mysql 
```

> On the MySQL prompt, run the following commands (**change the user/password** to your own values):

Reviewer 2: ```suggestion
At the MySQL prompt, run the following commands (change `user/password` to your own values):
```

> <pre>

Reviewer 2: I'm not familiar with `<pre>` - is it typed like that in the command prompt/terminal?

Author: no, it's bad HTML 
should be replaced with ```

Reviewer: this `<pre>` is still not replaced.

CREATE DATABASE boards;
GRANT ALL on boards.* to <b>'boardsuser'@'localhost'</b> identified by '<b>boardsuser-password</b>';

</pre>

Exit the mysql-prompt:
```
exit
```

> Edit the Focalboard config.json:

Reviewer 2: ```suggestion
Edit the Focalboard `config.json`:
```

```
nano /opt/focalboard/config.json
```

Change the dbconfig setting to use the MySQL database you created:
```
"dbtype": "mysql",

> "dbconfig": "mysql://boardsuser:boardsuser-password@tcp(127.0.0.1:3306)/boards",

Reviewer 3: Accoding to my test & #301 this shoud be:
```
"dbconfig": "boardsuser:boardsuser-password@tcp(127.0.0.1:3306)/boards",
```

Author: Indeed! Thanks for noticing
